packages/sitecore-jss/src/test-data/personalizeData.ts

```diff
@@ -0,0 +1,157 @@
+﻿export const layoutData = {
+  sitecore: {
+    context: {
+      pageEditing: false,
+      site: { name: 'JssNextWeb' },
+      visitorIdentificationTimestamp: 1038543,
+      language: 'en',
+    },
+    route: {
+      name: 'landingpage',
+      placeholders: {
+        'jss-main': [
+          {
+            uid: '0b6d23d8-c50e-4e79-9eca-317ec43e82b0',
+            componentName: 'ContentBlock',
+            dataSource: 'e020fb58-1be8-4537-aab8-67916452ecf2',
+            fields: { content: { value: '' }, heading: { value: 'Default Content' } },
+            experiences: {
+              mountain_bike_audirnce: {
```

Comment:
Still see lots of duplicates of these experience objects (`mountain_bike_audience` and `city_bike_audience`). Couldn't we create a const for each and reuse?